source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -0,0 +1,155 @@
+.. _swupd-mixer-integration:
+
+Mixer tool enhancement
+######################
+
+Mixing means composing an operating system for a specific use case. The
+default build of |CLOSIA| provides options to install bundles for various
+server capabilities. If you want to supplement the default OS with
+functionality from other OS distributions, you can manually compose an OS
+using :ref:`the mixer tool <mixer>`.
+
+Software updater release 19900 integrates the mixer tool with the
+updater (:command:`swupd`) and provides a simpler way to mix your OS.
+
+Process overview
+****************
+
+To add a package that does not exist in the current |CL| bundle, perform the following:
+
+#. Create custom packages and bundles using the :file:`swupd-add-pkg` script.
+
+#. Add your desired content with the commands:
+
+   .. code-block:: bash
+
+      sudo swupd update
+      sudo swupd bundle-add <new_bundle>
+
+The upstream content is pulled from the normal URL and local content is added
+from a local update store.
+
+.. note::
+
+   The integrated mixer tool adds non-unique packages and replaces upstream
+   content with a local version. However, the updater exits if it finds
+   conflicts before creating the local content. You can override this behavior
+   by passing the :command:`--allow-mix-collisions` option.
+
+
+Prerequisites
+*************
+
+To use the integrated swupd mixer, you need a recent image of
+|CLOSIA| with the mixer bundle installed. If you do not have it already,
```

Comment:
I think this should say if you do not have the mixer bundle already**
To me at first read-through it sounds like if you do not have a recent image of Clear installed... etc. Can fix this in a later commit after we merge this though.